python/GangaDirac/test/Unit/Utilities/TestDiracUtilitiesExternals.py

```diff
@@ -0,0 +1,78 @@
+import os
```

Comment:
I'm moving these tests as they don't contribute anything DIRAC specific here but these tests highlight again the need for testing that the tests will execute without a proxy present.